client_end_user/shared/src/commonMain/kotlin/App.kt

```diff
@@ -0,0 +1,19 @@
+import androidx.compose.runtime.Composable
+import cafe.adriel.voyager.navigator.Navigator
+import cafe.adriel.voyager.transitions.SlideTransition
+import data.gateway.local.LocalConfigurationGateway
+import kotlinx.coroutines.runBlocking
+import org.koin.compose.getKoin
+import presentation.pickLanguage.PickLanguageScreen
+import resources.BeepBeepTheme
+
+@Composable
+fun App() {
+    val localConfigurationGateway = LocalConfigurationGateway(getKoin().get())
```

Comment:
there is an AppModelScreen u can get languged code instad of call gateway inside app check it in - presentetion -> app dirctory